src/products/components/HalfSectionProduct/HalfSectionProduct.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC } from "react";
+import { HalfSectionProductStyled } from "./HalfSectionProduct.styled";
+import { selectProductsBySlug } from "../../slice";
+import { useSelector } from "react-redux";
+import { Grid, Typography, Button, Box } from "../../../_shared";
+
+interface Props {
+  product: string;
+  direction: "right" | "left";
+}
+
+export const HalfSectionProduct: FC<Props> = ({ product, direction }) => {
+  const productData = useSelector((state) =>
+    selectProductsBySlug(state, product)
+  );
+
+  return (
+    <HalfSectionProductStyled>
+      <Grid
+        container={true}
+        alignItems={"center"}
+        justifyContent={"center"}
+        direction={direction === "right" ? "row" : "row-reverse"}
+      >
+        <Grid item={true} xs={12} md={6} className="img-container">
+          <Box
+            mr={direction === "right" ? 5 : 0}
+            ml={direction === "right" ? 0 : 5}
+          >
+            <img
+              src={`/assets/product-${productData.slug}/desktop/image-product.jpg`}
```

Comment:
Maybe this image address can be added directly to product data. Moving images maybe to a cms or in another folder make you change de component itself